src/NServiceBus.RabbitMQ/Receiving/MessagePump.cs

```diff
@@ -174,74 +174,53 @@ void Connection_ConnectionShutdown(object sender, ShutdownEventArgs e)
 
                 if (rejectMessage)
                 {
-                    await RejectMessage(channel, message.DeliveryTag, messageId).ConfigureAwait(false);
+                    await RejectMessage(channel, message.DeliveryTag).ConfigureAwait(false);
                 }
                 else
                 {
-                    await AcknowledgeMessage(channel, message.DeliveryTag, messageId).ConfigureAwait(false);
+                    await AcknowledgeMessage(channel, message.DeliveryTag).ConfigureAwait(false);
                 }
             }
+            catch (AlreadyClosedException ex)
+            {
+                Logger.Warn($"Attempt to acknowledge/reject message {messageId} failed because the channel was closed. The message will be requeued.", ex);
```

Comment:
I didn't do that originally since I'd need to add additional try blocks around each call, but you're right that preserving the ack/reject distinction in the log would be useful.